module/zfs/dsl_scan.c

```diff
@@ -68,18 +68,6 @@
  * queue and issue them out of logical order and in a way that most benefits
  * physical disks (LBA-order).
  *
- * This sorting algorithm is subject to limitations. We can't do this with
- * blocks that are non-leaf, because the scanner itself depends on these
- * being available ASAP for further metadata traversal. So we exclude any
- * block that is bp_level > 0. Fortunately, this usually represents only
- * around 1% of our data volume, so no great loss.
- *
```

Comment:
The paragraph above remains accurate, even if the one below isn't, right?  Should it be kept?